CMakeLists.txt

```diff
@@ -215,6 +215,10 @@ option(SWIFT_STDLIB_ENABLE_RESILIENCE
     "Build the standard libraries and overlays with resilience enabled; see docs/LibraryEvolution.rst"
     FALSE)
 
+option(SWIFT_STDLIB_ENABLE_REFLECTION_METADATA
+    "Build the standard libraries and overlays with remote reflection metadata; see docs/proposals/RemoteMirrors.rst"
+    FALSE)
+
```

Comment:
@nadavrot Looks like a modest increase. Note that this is data size, not code size, and it should in theory be true const (but we should check):

    -rwxr-xr-x  1 slava  staff  4298224 Apr 12 15:02 /Users/slava/src/new/build/Ninja-ReleaseAssert+swift-DebugAssert/swift-macosx-x86_64/lib/swift/macosx/libswiftCore.dylib
    -rwxr-xr-x  1 slava  staff  4347376 Apr 12 12:52 /Users/slava/src/new/build/Ninja-ReleaseAssert+swift-DebugAssert/swift-macosx-x86_64/lib/swift/macosx/libswiftCore.dylib